pkgs/applications/networking/p2p/mldonkey/default.nix

```diff
@@ -1,37 +1,31 @@
-{ lib, stdenv, fetchurl, ocamlPackages, zlib, bzip2, ncurses, file, gd, libpng, libjpeg }:
+{ lib, stdenv, fetchurl, ocamlPackages, zlib }:
 
-stdenv.mkDerivation ({
-  name = "mldonkey-3.1.6";
+stdenv.mkDerivation rec {
+  pname = "mldonkey";
+  version = "3.1.7-2";
 
   src = fetchurl {
-    url = "https://github.com/ygrek/mldonkey/releases/download/release-3-1-6/mldonkey-3.1.6.tar.bz2";
-    sha256 = "0g84islkj72ymp0zzppcj9n4r21h0vlghnq87hv2wg580mybadhv";
+    url = "https://ygrek.org/p/release/mldonkey/mldonkey-${version}.tar.bz2";
```

Comment:
Any reason why not use the github link? It is the same as far as I can tell...
https://github.com/ygrek/mldonkey/releases/download/release-3-1-7-2/mldonkey-3.1.7-2.tar.bz2

Reply:
That is the URL listed in opam-repository which I used…